pages/introduction/database.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Database
+parent: Introduction
+nav_order: 4
+permalink: /introduction/database
+---
+
+# PhyloFisher’s Provided Database
+
+### PhyloFisher’s Provided Database Is...
+The PhyloFisher package is equipped with a manually curated database of 240 protein coding genes from up to 304 eukaryotes that can be used for phylogenomic analyses of eukaryotic organisms. Details on the genes and taxa included in the database can be found currently at the below link. A metadata file (metadata.tsv) that provides information on the proteomes used in database construction is included with the database and can be found in the directory database/. In addition to a single ortholog of each of the databases’s 240 genes, a database of each gene’s paralogs is maintained.
```

Comment:
Is the link mentioned present some place? I don't see it.

All mentions of the metadata.tsv should be substituted for something along the lines of "the contents of the database can be viewed once downloaded using the tool explore_database.py provided with PhyloFisher"

Reply:
Not sure exactly what this is supposed to be linking to. There isn't a link present in the manual either.

Reply:
In rereading it, I think we can just delete that sentence and updating the following one covers what we need to say.